lib/modules/commands/responses.rb

```diff
@@ -25,6 +25,16 @@ module Actions
         event << "#{event.user.display_name} #{response} boops #{thing}"
         event.message.delete
       end
+      
+      command(:colour,
+             description: 'Returns matching colours',
+             usage: "#{BOT.prefix} colour") do |event|
+        thing = thing.join(' ')
+        response = Database::Response.where(key: 'colour')
+                                     .all.sample.response
+        event << "#{event.user.display_name} #{response}"
+        event.message.delete
```

Comment:
I would prefer for us not to delete invoking messages for things like this where data is returned. For actions it makes sense, for this, it doesn't.